cloud/pkg/devicecontroller/manager/common.go

```diff
@@ -44,3 +44,8 @@ func (c *CommonResourceEventHandler) OnDelete(obj interface{}) {
 func NewCommonResourceEventHandler(events chan watch.Event) *CommonResourceEventHandler {
 	return &CommonResourceEventHandler{events: events}
 }
+
+// GetResourceID return resource ID
+func GetResourceID(namespace, name string) string {
+	return namespace + "-" + name
```

Comment:
```suggestion
	return namespace + "/" + name
```
We should use `/` here, because the namesapce or name maybe include `-`, then you can't split ns/name from the key.

Ref in k8s: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/tools/cache/object-names.go#L44

Reply:
Done @fisherxu